astro/src/content/docs/apis/identity-providers/_identity-provider-login-response-body.mdx

```diff
@@ -14,10 +14,11 @@ _Response Codes_
 | 204  | The login is pending action by the end user. Wait and try the request again.                                                                                                                                                                                                                                                                                                                                                                   |
 | 212  | The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `FormField`, the response will contain the <InlineField>emailVerificationId</InlineField> that was generated for the user.                                                                                                                                                  |
 | 213  | The user's registration has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `FormField`, the response will contain the <InlineField>registrationVerificationId</InlineField> that was generated for the user.<br/>Prior to version `1.27.0`, this status code was not returned, and you will see a `200` instead.                                        |
-| 242  | The user was authenticated successfully. The user has two factor authentication enabled. Since version `1.42.0`, this status code is also returned when two factor authentication is required. The response will contain the `twoFactorId` to be used on the [Complete Two Factor Authentication](/docs/apis/login#complete-multi-factor-authentication).<br/><JSON title="Example Response JSON" src="login/login-two-factor-response.json" /> |
+| 232  | The user is completing a login with an Identity Provider configured to use the `Create a pending link` linking strategy and this user does not have a link established.  The API response will only contain the `pendingIdPLinkId` and, possibly, `threatsDetected`. A call to the Link API's [Complete a Pending Link](/docs/apis/identity-providers/links#complete-a-pending-link) endpoint is required. Once the user is linked, calling this API again will allow login to succeed.|
```

Comment:
@jobannon any reason we can't put in a sample JSON response? I thought I'd asked for that but maybe I mis-remember?

Reply:
You'd have to create the json under `astro/src/content/json/login` but then could add it just like the response is added at the bottom of the `242` response.

Reply:
@mooreds Ah, misunderstood.